modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/topology/PartitionsEvictManager.java

```diff
@@ -217,13 +256,31 @@ private void showProgress() {
         if (U.millisSinceNanos(lastShowProgressTimeNanos) >= evictionProgressFreqMs) {
             int size = evictionQueue.size() + 1; // Queue size plus current partition.
 
-            if (log.isInfoEnabled())
-                log.info("Eviction in progress [permits=" + permits+
+            if (log.isInfoEnabled()) {
+                log.info("Eviction in progress [permits=" + permits +
                     ", threads=" + threads +
                     ", groups=" + evictionGroupsMap.keySet().size() +
                     ", remainingPartsToEvict=" + size + "]");
 
-            evictionGroupsMap.values().forEach(GroupEvictionContext::showProgress);
+                evictionGroupsMap.values().forEach(GroupEvictionContext::showProgress);
+
+                if (!logEvictPartByGrps.isEmpty()) {
+                    StringJoiner evictPartJoiner = new StringJoiner(", ");
+
+                    logEvictPartByGrps.forEach((grpId, evictParts) -> {
+                        CacheGroupContext grpCtx = cctx.cache().cacheGroup(grpId);
+                        String grpName = (nonNull(grpCtx) ? grpCtx.cacheOrGroupName() : null);
+
+                        String partByReasonStr = partByReasonStr(evictParts);
+
+                        evictPartJoiner.add("[grpId=" + grpId + ", grpName=" + grpName + ", " + partByReasonStr + ']');
```

Comment:
Please don't use code style abbreviations in string messages, it makes logs hard to read.